### PR TITLE
Fix cylc install help examples

### LINKED ISSUE #4186: Fix install examples
Ref https://github.com/cylc/cylc-flow/pull/4161#issuecomment-814981966

### PR DESCRIPTION
**Pull request by MetRonnie**
These changes close #4186 

Would be good to get this in for 8.0b1

**Requirements check-list**
- [x] I have read `CONTRIBUTING.md` and added my name as a Code Contributor.
- [x] Contains logically grouped changes (else tidy your branch by rebase).
- [x] Does not contain off-topic changes (use other PRs for other changes).
- [x] Does not need tests (can't test these automatically).
- [x] Appropriate change log entry included.
- [x] No documentation update required.
- [x] No dependency changes.

### CODE REVIEW

**Review by datamel — APPROVED** [review]
Thanks @MetRonnie. Tested as working.

File: cylc/flow/scripts/install.py [hunk 1]
```diff
@@ -56,12 +56,12 @@
   # run directory ~/cylc-run/fido/run1
   $ cylc install --flow-name=fido
 
-  # Install $PWD/bunny/rabbit/flow.cylc as "bunny/rabbit", with run directory
-  # ~/cylc-run/bunny/rabbit/run1
+  # Install $PWD/bunny/rabbit/flow.cylc as "rabbit", with run directory
+  # ~/cylc-run/rabbit/run1
   $ cylc install --directory=bunny/rabbit
```
Comment (wxtim) [inline cylc/flow/scripts/install.py:61]: Naïve test
```bash
> cd ~/cylc-src
> mkdir -p bunny/rabbit
> touch bunny/rabbit/flow.cylc
> cylc install --directory=bunny/rabbit
INSTALLED cylc-src from bunny/rabbit -> /home/h02/tpilling/cylc-run/**cylc-src/run1**
> cylc install --directory=bunny/rabbit
```

**Review by wxtim — CHANGES_REQUESTED** [review]
I tried following instructions and it didn't work as expected...

File: cylc/flow/scripts/install.py (hunk 1 above)
Comment (MetRonnie) [reply cylc/flow/scripts/install.py:61]: Hmm, that was supposed to be fixed by #4161, I'll look into it

Comment (MetRonnie) [reply cylc/flow/scripts/install.py:61]: Cannot reproduce, I get
```
INSTALLED rabbit from bunny/rabbit -> ~/cylc-run/rabbit/run1
```

**Review by kinow — APPROVED** [review]
Worked for me too.

```bash
(venv) kinow@ranma:~/Development/python/workspace/cylc-flow$ cd ~/cylc-suites
(venv) kinow@ranma:~/cylc-suites$ mkdir -p bunny/rabbit
(venv) kinow@ranma:~/cylc-suites$ touch bunny/rabbit/flow.cylc
(venv) kinow@ranma:~/cylc-suites$ cylc install --directory=bunny/rabbit
INSTALLED rabbit from bunny/rabbit -> /home/kinow/cylc-run/rabbit/run1
```

File: cylc/flow/scripts/install.py (hunk 1 above)
Comment (hjoliver) [reply cylc/flow/scripts/install.py:61]: Same, perhaps you hadn't checked out this branch @wxtim?

**Review by hjoliver — APPROVED** [review]

**Pull request merged**